sssom/util.py

```diff
@@ -885,13 +885,14 @@ def to_mapping_set_dataframe(doc: MappingSetDocument) -> MappingSetDataFrame:
     # convert MappingSetDocument into MappingSetDataFrame
     ###
     data = []
-    for mapping in doc.mapping_set.mappings:
-        mdict = mapping.__dict__
-        m = {}
-        for key in mdict:
-            if mdict[key]:
-                m[key] = mdict[key]
-        data.append(m)
+    if doc.mapping_set.mappings is not None:
```

Comment:
@cthoyt , does this makes sense. This is the fix for the error:

`Item "None" of "Union[Dict[Any, Any], Mapping, List[Union[Dict[Any, Any], Mapping]], None]" has no attribute "__iter__" (not iterable)`

Reply:
we should really fix #145 before doing more on here otherwise we're going to get merge problems. I'll comment on this one after we finish 145